### PR TITLE
Pass SendKeys state by ref

### LINKED ISSUE #6666: SendKeys Grouping fails
* .NET Core Version: 
6.0

* Have you experienced this same bug with .NET Framework?: 
NO

**Problem description:**

SendKeys.Send("^(a)^(c)") triggers exception.

ArgumentException
Exception:SendKeys string '^(a)^(c)' is not valid.

**Expected behavior:**
 Sends Ctrl-A then Ctrl-C

**Minimal repro:**
SendKeys.Send("^(a)^(c)")

### PR DESCRIPTION
**Pull request by JeremyKuhne**
Regression introduced in 90f7c1dd955c51c904efd496731a2394cacf578f.

Fixes #6666

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/6710)

### CODE REVIEW

**Review by dreddy-work — APPROVED** [review]

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/SendKeysTests.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Windows.Forms.Tests
+{
+    public class SendKeysTests
+    {
+        [WinFormsFact(Skip = "This test depends on focus and should be run manually.")]
```
Comment (RussKie) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/SendKeysTests.cs:11]: Please move this test into https://github.com/dotnet/winforms/tree/main/src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests test suite, those test interact with keyboard, mouse, etc.

**Review by RussKie — CHANGES_REQUESTED** [review]
👍

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/SendKeysTests.cs (hunk 1 above)
Comment (JeremyKuhne) [reply src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/SendKeysTests.cs:11]: @RussKie I will follow up with a separate change to move it. I want this to be able to cleanly port back to 6.0.

**Comment by JeremyKuhne** [comment]
I'll be moving the test later as mentioned: https://github.com/dotnet/winforms/pull/6710#discussion_r809695722.

**Pull request merged**